src/main/java/com/minecolonies/coremod/util/BlockUtils.java

```diff
@@ -400,4 +400,37 @@ public static boolean isPathBlock(final Block block)
     {
         return block == Blocks.GRAVEL || block == Blocks.STONEBRICK;
     }
+
+    /**
+     * Getter for a Block from an itemStack.
+     * @param stack the incoming itemStack
+     * @return the outgoing block
+     */
+    public static Block getBlockFromItemStack(final ItemStack stack)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "getBlockFromItemStack" is 14 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)